docs/azure_arc_jumpstart/azure_arc_app_svc/aks/aks_container_apps_arm_template/_index.md

```diff
@@ -0,0 +1,291 @@
+
+    ![Screenshot showing PowerShell logon script run](./11.png)
+
+    ![Screenshot showing PowerShell logon script run](./12.png)
+
+    ![Screenshot showing PowerShell logon script run](./13.png)
+
+    ![Screenshot showing PowerShell logon script run](./14.png)
+
+    ![Screenshot showing PowerShell logon script run](./15.png)
+
+    ![Screenshot showing PowerShell logon script run](./16.png)
+
+    ![Screenshot showing PowerShell logon script run](./17.png)
+
+    ![Screenshot showing PowerShell logon script run](./18.png)
+
+  Once the script finishes it's run, the logon script PowerShell session will be closed, the Windows wallpaper will change, and the Azure Container Apps deployed on the cluster will be ready.
+
+    ![Screenshot showing desktop wallpaper change](./19.png)
+
+- Since this scenario is deploying both the Azure Container Apps, you will also notice additional, newly deployed Azure resources in the resources group. The important ones to notice are:
+
+  - **Azure Arc-enabled Kubernetes cluster** - Azure Arc-enabled Kubernetes are used to deploy the Azure Containers Apps[cluster extension](https://docs.microsoft.com/azure/azure-arc/kubernetes/conceptual-extensions), as well as using Azure Arc [Custom locations](https://docs.microsoft.com/azure/azure-arc/kubernetes/conceptual-custom-locations).
+
+  - **Custom location** - Provides a way for tenant administrators to use their Azure Arc-enabled Kubernetes clusters as a target location for deploying Azure services.
+
+  - **Container Apps Connected Environment**- Container Apps connected environment enables configuration common across apps but not related to cluster operations. Conceptually, it's deployed into the custom location resource, and app developers create apps into this environment.
+
+  - **Container App** - Azure Container Apps manages the details of Kubernetes and container orchestration for you. Containers in Azure Container Apps can use any runtime, programming language, or development stack of your choice.
+  
+  ![Screenshot showing additional Azure resources in the resource group](20.png)
+
+- In this scenario, **ASP.NET Core Front-end and 2 Back-end APIs on Azure Container Apps** was deployed. To open the deployed web application in your web browser, simply click the Container App resource and the Application URL.
+
+  ![Screenshot showing Store Container App in a resource group](./21.png)
+
+  ![Screenshot showing the web application URL](./22.png)
+
+  ![Screenshot showing the web application open in a web browser](./23.png)
+
+- Optionally, you can review the Dapr configuration for the Container apps by navigating to the Dapr setting.
```

Comment:
```suggestion
- Optionally, you can review the [Dapr](https://learn.microsoft.com/azure/container-apps/dapr-overview?tabs=bicep1%2Cyaml) configuration for the Container apps by navigating to the Dapr setting.
```

Reply:
In the overview section, when introducing the scenario. i suggest calling out the Dapr integration